.github/workflows/jupyter.yaml

```diff
@@ -65,8 +63,8 @@ jobs:
         export PATH="$PWD"/vast/bin:"$PATH"
         nohup vast start &
         while ! lsof -i ":42000"; do sleep 1; done
-        vast import zeek -r vast-jupyter-demo/M57-day11-18-conn.log
-        vast import suricata -r vast-jupyter-demo/M57-day11-18.json
-        vast import pcap -r vast-jupyter-demo/M57-2009-day11-18.trace
+        vast import -r vast-jupyter-demo/M57-day11-18-conn.log zeek
+        vast import -r vast-jupyter-demo/M57-day11-18.json suricata
+        vast import -r vast-jupyter-demo/M57-2009-day11-18.trace pcap
```

Comment:
:pray:

Reply:
We should think about running this workflow every so often. The current triggers are apparently not sufficient for CI.